lib/elixir/test/elixir/module_test.exs

```diff
@@ -310,4 +316,33 @@ defmodule ModuleTest do
       assert Module.definitions_in(__MODULE__, :defp) == []
     end
   end
+
+  describe "make_overridable/2 arguments" do
```

Comment:
Check lib/elixir/test/elixir/kernel/overridable_test.exs. We don't need to add the success test because there are many there. You can add a test for the error in make_overridable though.